docs/Migrating.md

```diff
@@ -8,6 +8,7 @@
 #### Steps to Migrate
 * `UnitySDK/Assets/ML-Agents/Scripts/Communicator.cs` and its class `Communicator` have been renamed to `UnitySDK/Assets/ML-Agents/Scripts/ICommunicator.cs` and `ICommunicator` respectively.
 * The `SpaceType` Enums `discrete`, and `continuous` have been renamed to `Discrete` and `Continuous`.
+* We have removed the `Done` call as well as the capacity to set `Max Steps` on the Academy. Therefore, it is now impossible to call an AcademyReset from C# (only possible from Python). If you want to reset the simulation after a fixed number of steps, or when an event in the simulation occurs, we recommend looking at our multi-agent example environments (such as BananaCollector). In our examples, groups of Agents can be reset through an "Area" that can reset groups of Agents.
```

Comment:
> Therefore, it is now impossible to call an AcademyReset from C# (only possible from Python). 

It's still possible for the user to call it from "game" code, right? Maybe rephrase as something like "Therefore an AcademyReset will never be triggered from C# (only from Python)"?